cadc-util/src/main/java/ca/nrc/cadc/net/FileContent.java

```diff
@@ -0,0 +1,31 @@
+package ca.nrc.cadc.net;
+
+/**
+ * Created by jeevesh on 2018-10-17.
+ */
+public class FileContent {
+
+    String content;
+    String contentType;
+
+    public FileContent(String content) {
+        this.content = content;
+    }
+
+    public String getContent() {
```

Comment:
For a class like this I would not have a setContent method (immutable) and I would throw an IllegalArgumentException from the constructor if the content is null. Then the content field can be private and final.

Generally, if a field is allowed to be null it might as well be public and have no methods (contentType).

Reply:
I don't know which way to take the FileContent class then. Both fields mandatory? contentType not mandatory? That's the question that needs answering before I can put the code one way or the other.

I set fileContent up to have both be mandatory. Then probably made it unclear by putting a check on it's value in the HttpPost code that prints out the multipart headers?

Reply:
Both are mandatory, so:

1. Check for null values in constructor
2. Make the attributes private
3. Have a getter for each but no setter

Reply:
For the code that is dependent on this change, the content type is mandatory. (It's used to filter the stream input to one reader or another. I'm not sure what the requirement would be for other code (ie potentially the "tap" services) that would use this code path? 

I can make contentType optional for FileContent, then put a check in the InlineContentHandler...